token-validation-spring-test/src/main/java/no/nav/security/token/support/spring/test/MockLoginController.java

```diff
@@ -60,4 +59,41 @@ public Cookie addCookie(
         }
         return cookie;
     }
+
+    @Unprotected
+    @PostMapping("/cookie")
```

Comment:
suggest adding the issuerId as a PathVariable, e.g. `@PostMapping("/cookie/{issuerid}")`

Reply:
done